### DISCUSSION TITLE
Can I deploy a Golang API

### DISCUSSION THREAD

**Discussion by PHTremor** [deta/cloud-docs#592] [Help]
I've seen Go examples on the site, connecting with the base. is it possible to deploy an app written in Go on Deta?

**Comment by lemonyte** [comment]
Micros on the current platform (https://deta.sh) only support Python and Node.js runtimes. The new Space platform (https://deta.space) supports Go and many other runtimes. You can apply for alpha access [here](https://alpha.deta.space).

Duplicate of #580.
